File: 4A - Watermelon/main.cpp
/*
    https://codeforces.com/problemset/submission/4/225039976
    GNU C++17 - 30 ms -	0 KB
*/

#include<iostream>
using namespace std;

int main(int argc, char const *argv[])
{
    int w;
    cin >> w;
    if ((w % 2 == 0) && (w > 2)) {
        cout << "YES";
    } else {
        cout << "NO";
    }
    return 0;
}

File: 4A - Watermelon/main.php
<?php
// https://codeforces.com/problemset/submission/4/225041775
// PHP - 92 ms - 16 KB

$file = fopen('php://stdin', 'r');
$w = fgets($file);
fclose($file);

if(($w % 2 == 0) && ($w > 2)) {
    echo("YES");
} else {
    echo("NO");
}

?> 